assets/js/app.js

```diff
@@ -229,3 +230,37 @@ function setupPrint() {
     window.print();
   });
 }
+
+function setupDatePicker() {
+  if (!$.prototype.flatpickr) { return; }
+
+  $(".datepicker").each(function() {
+    let options = {
+      enableTime: true,
+      time_24hr: true,
+      altInput: true,
+      altFormat: "F j, Y H:i \\U\\T\\C",
+      dateFormat: "Y-m-d H:i",
+      monthSelectorType: "static",
+      parseDate(dateStr, dateFormat) {
```

Comment:
Who calls this? Are we overriding default parseDate function that comes with flatpickr?

Reply:
Yeah. It's from flatpickr.
https://flatpickr.js.org/instance-methods-properties-elements/#parsedatedatestr-dateformat